Elastic API check fix

Zenidd opened the pull request:
Hi team,

The current Elasticsearch API test after opendistro install fails on clusters with yellow status. This PR sets yellow status as a valid one for task success.

Greetings,

JP

xr09 commented on roles/opendistro/opendistro-elasticsearch/tasks/main.yml:
```diff
@@ -64,7 +64,7 @@
       timeout: 4
     register: _result
-    until: (  _result.json is defined) and (_result.json.status == "green")
+    until: (  _result.json is defined) and (_result.json.status == "green" or _result.json.status == "yellow" )
```
Should we also check for single node condition here?

```yaml
until: (  _result.json is defined) and (_result.json.status == "green" or (_result.json.status == "yellow" and single_node) )
```

Zenidd replied:
I agree, this way we ensure that the check is successful when the health of the cluster is yellow only for single-node configurations.

Fixed here d23feddedec43a1ae37b956ec66241c62249fb7d

xr09 reviewed (approved):
LGTM!

neonmei reviewed (approved):
Luce bien :+1:

neonmei commented:
Un detallín extra, es que los `and` del until se pueden poner como array (ansible >2.8.algo), algo asi:

```yaml
      until:
        - _result.json is defined
        - _result.json.status == "green" or  _result.json.status == "yellow" and single_node
```
y el block que arranca en la línea 8 necesita un nivel de identación más (2 espacios)

Zenidd commented:
Just replaced the until statement syntax with your suggestion @neonmei.  I added parenthesis in the conditional checking the cluster status to ensure legibility. Fixed on af948dff018c380c224abe651362143a6bb53969
